tests/strategies/test_mapping.py

```diff
@@ -0,0 +1,56 @@
+"""Tests mapping strategy."""
+import dlite
+from tripper import EMMO, MAP, Namespace
+
+from oteapi_dlite.strategies.mapping import (
+    DLiteMappingConfig,
+    DLiteMappingStrategy,
+)
+
+FORCES = Namespace("http://onto-ns.com/meta/0.1/Forces#")
+ENERGY = Namespace("http://onto-ns.com/meta/0.1/Energy#")
+
+# Test without prefixes
+config = DLiteMappingConfig(
+    mappingType="mappings",
+    triples=[
+        (FORCES.forces, MAP.mapsTo, EMMO.Force),
+        (ENERGY.energy, MAP.mapsTo, EMMO.PotentialEnergy),
+    ],
+)
+coll = dlite.Collection()
+session = {"collection_id": coll.uuid}
+
+mapper = DLiteMappingStrategy(config)
+mapper.initialize(session)
+mapper.get(session)
+
+relations = set(coll.get_relations())
+assert (FORCES.forces, MAP.mapsTo, EMMO.Force) in relations
+assert (ENERGY.energy, MAP.mapsTo, EMMO.PotentialEnergy) in relations
+
+
+# Test with prefixes
+config2 = DLiteMappingConfig(
+    mappingType="mappings",
+    prefixes={
+        "f": "http://onto-ns.com/meta/0.1/Forces#",
+        "e": "http://onto-ns.com/meta/0.1/Energy#",
+        "map": str(MAP),  # __FIXME__: prefixes should accept a Namespace
+        "emmo": str(EMMO),
+    },
+    triples=[
+        ("f:forces", "map:mapsTo", "emmo:Force"),
+        ("e:energy", "map:mapsTo", "emmo:PotentialEnergy"),
+    ],
+)
+coll2 = dlite.Collection()
+session2 = {"collection_id": coll2.uuid}
+
+mapper2 = DLiteMappingStrategy(config2)
+mapper2.initialize(session2)
+mapper2.get(session2)
+
+relations2 = set(coll2.get_relations())
```

Comment:
Why a set?

Reply:
If this should be a set, check the length does not change.

Reply:
> Why a set?

Because, in general, the two following containment tests are much more efficient when applied to a set than a list.
The return value of `coll2.get_relations()` is an iterator, so if we want to reuse the output we anyway have to convert it to a sequence.

Reply:
Okay. Would be good to check that this works if `coll2.get_relations()` returns equal values throughout as well. This can be checked by simply ensuring that `len(coll2.get_relations()) == len(relations2)`. Unless it's ensured that all objects returned by `get_relations()` will always resolve as unique for the `set()` uniqueness determination logic.

Reply:
Ok, added the test. len() does not accept an iterator, so it has to be writted as `assert len(list(coll2.get_relations())) == len(relations2)`